server/datastore/mysql/mdm_test.go

```diff
@@ -166,6 +168,50 @@ func testMDMCommands(t *testing.T, ds *Datastore) {
 	require.Equal(t, winCmd.TargetLocURI, cmds[1].RequestType)
 	require.Equal(t, "Pending", cmds[1].Status)
 
+	// filter by host Identifier
+	identifiers := map[string]string{
+		windowsH.Hostname:       winCmd.CommandUUID,
+		*windowsH.OsqueryHostID: winCmd.CommandUUID,
+		*windowsH.NodeKey:       winCmd.CommandUUID,
+		windowsH.UUID:           winCmd.CommandUUID,
+		windowsH.HardwareSerial: winCmd.CommandUUID,
+		macH.Hostname:           appleCmdUUID,
+		*macH.OsqueryHostID:     appleCmdUUID,
+		*macH.NodeKey:           appleCmdUUID,
```

Comment:
Just FYI: @jacobshandling is removing node key from being a host identifier in this PR: https://github.com/fleetdm/fleet/pull/20220